### PR TITLE
Adding a variable to indicate if the cleanup Artifactory should run

### PR DESCRIPTION
**Pull request by samuel-rubin**
The cleanup job will only be called if the variable is set to true

[skip ci]
Signed-off-by: Samuel Rubin <samuel.rubin@ibm.com>

### CODE REVIEW

**Comment by samuel-rubin** [comment]
If the variable is set to true and the cleanup job is not configured, the console output is: 
```
Passing ARTIFACTORY_MANUAL_CLEANUP:'true'
[Pipeline] build
[Pipeline] echo
The Cleanup_Artifactory job is not availible
```
If the variable is set to false, the console output is: 
```
[Pipeline] echo
Passing ARTIFACTORY_MANUAL_CLEANUP:'false'
```
If the variable is not set, the console output is:
```
[Pipeline] echo
Passing ARTIFACTORY_MANUAL_CLEANUP:'null'
```

**Comment by samuel-rubin** [comment]
@AdamBrousseau for review

File: buildenv/jenkins/common/pipeline-functions [hunk 1]
```diff
@@ -407,7 +409,7 @@ def get_test_target_names() {
 
 def get_build_job_name(spec, version, identifier) {
     id = convert_build_identifier(identifier)
-    return "Build_JDK${version}_${spec}_${id}"
+    return "Sam-Build_JDK${version}_${spec}_${id}"
```
Comment (AdamBrousseau) [inline buildenv/jenkins/common/pipeline-functions]: `Sam-`

Comment (samuel-rubin) [reply buildenv/jenkins/common/pipeline-functions]: Whoops. Fixed

**Pull request merged**